tmt/utils.py

```diff
@@ -1502,6 +1517,40 @@ def copytree(
     return dst
 
 
+def get_full_metadata(fmf_tree_path: str, node_path: str) -> Any:
```

Comment:
yeah, it's not used, it was pulled from beakerlibs pruning as a base for the pruning itself as it will actually be necessary there, but I wanted to have it here as it will likely be necessary for tests pruning as well

Reply:
Ok, thanks for clarification. The one-liner part still remains but let's keep it and see if the function is to be extended in the future.